### ISSUE TITLE
Role göre task

### ISSUE THREAD

**Issue by Lofiwins** [aloshai/discord-moderation-bot#3]
İD'si girilen rolde ki herkese görev hakkı verme örnek bir public sunucuda yetkililer yetki atlamak için 30 sat ses 150 mesaj mesala o roldekiler yapıcak yapıncada rol vericek mesela fonksiyonel olabilir ödül ayarlama gibi daha detaylı istersen söylerim mert 
Discord: Erdem#1752

**Comment by aloshai** [comment]
Evet, görevi yaptıktan sonra bulunduğun yetkinin bir üstüne çıkma gibi bir şey ekleyebilirim :) beğendim yakın zamanda ilgileneceğim

**Comment by Lofiwins** [comment]
> Evet, görevi yaptıktan sonra bulunduğun yetkinin bir üstüne çıkma gibi bir şey ekleyebilirim :) beğendim yakın zamanda ilgileneceğim

<3
